stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2015 Mikel Arregi - AvanzOSC
+# Copyright 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare
+
+
+class AssignManualQuants(models.TransientModel):
+    _name = 'assign.manual.quants'
+
+    @api.multi
+    @api.constrains('quants_lines')
+    def check_qty(self):
+        precision_digits = self.env[
+            'decimal.precision'].precision_get('Product Unit of Measure')
+        move = self.env['stock.move'].browse(self.env.context['active_id'])
+        for record in self.filtered('quants_lines'):
+            if float_compare(record.lines_qty, move.product_qty,
+                             precision_digits=precision_digits) > 0:
+                raise UserError(
+                    _('Quantity is higher than the needed one'))
+
+    @api.depends('quants_lines', 'quants_lines.qty')
+    def _compute_qties(self):
+        move = self.env['stock.move'].browse(self.env.context['active_id'])
```

Comment:
I think checking active_id in a compute method is not correct

Reply:
Well, as this is always executed on a wizard popup, is tolerable, but yes, you can add a helper field `move_id` for storing the move reference, which is filled by default with the active_id.

Reply:
@rousseldenis 
If I change this line for self.move_id, I get move = stock.move() and quantities are calculated incorrectly. So I let this line as is.

Reply:
@cubells the idea is that `move_id` is filled on `default_get` method (or a direct default method).